staticjinja/dep_graph.py

```diff
@@ -0,0 +1,127 @@
+# -*- coding:utf-8 -*-
+
+"""
+Dependency graph for staticjinja
+"""
+
+
+class DepGraph(object):
+    """
+    A directed graph which will handle dependencies between sources in a site.
```

Comment:
Yes it's a remnant of the older try. If it seems this will get merged I'll try to remove all such things (I guess there are also a couple of variable names which could change).